### PR TITLE
DINOv2+Rein LoRA

### PR DESCRIPTION
**Pull request by Matnay**

### CODE REVIEW

File: robomimic/envs/env_robosuite.py [hunk 1]
```diff
@@ -136,14 +137,10 @@ def reset_to(self, state):
...
             self.reset()
-            robosuite_version_id = int(robosuite.__version__.split(".")[1])
-            if robosuite_version_id <= 3:
-                from robosuite.utils.mjcf_utils import postprocess_model_xml
-                xml = postprocess_model_xml(state["model"])
-            else:
-                # v1.4 and above use the class-based edit_model_xml function
-                xml = self.env.edit_model_xml(state["model"])
-            self.env.reset_from_xml_string(xml)
+            # ----- loading LIBERO model xml ----
```
Comment (SimarKareer) [inline robomimic/envs/env_robosuite.py:152]: What was this change about btw?

File: robomimic/models/base_nets.py [hunk 2]
```diff
@@ -540,7 +539,207 @@ def __repr__(self):
...
+        self._vit_model_class = vit_model_class
+        self._freeze = freeze
+        self._input_coord_conv = False
+        self._pretrained = False
+        self._lora_dim = lora_dim
+        self._patch_size = patch_size
+        self._out_indices = [7, 11, 15, 23],
+
+        self.preprocess = nn.Sequential(
+            transforms.Resize((294,294)),
```
Comment (SimarKareer) [inline robomimic/models/base_nets.py:581]: Did you write 294 or did you get this from somewhere.  Just wanted to make sure 294 was the native size that dino is expecting.  Also worth noting that we're squeezing a rectangular image into a square, but should be fine for now

File: robomimic/envs/env_robosuite.py (hunk 1 above)
Comment (Matnay) [reply robomimic/envs/env_robosuite.py:152]: This is not a change I made intentionally, might be due to a version mismatch. Reverting

File: robomimic/models/base_nets.py (hunk 2 above)
Comment (Matnay) [reply robomimic/models/base_nets.py:581]: This was the native size

**Pull request merged**